weldx/geometry.py

```diff
@@ -2328,6 +2329,26 @@ def __post_init__(self):
             if not self.triangles.ndim == 2:
                 raise ValueError("SpatialData triangulation must be a 2d array")
 
+    @staticmethod
+    def from_file(file_name: str) -> "SpatialData":
```

Comment:
`pathlib.Path` also seems to be supported by meshio so we might add that to the type hints

Reply:
yes. I am currently experimenting and writing the test ;)

Reply:
It looks like the library is leaking file handles. But using a sane OS this ain't a problem :p